test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -99,6 +99,7 @@ u256 EVMInstructionInterpreter::eval(
 	switch (_instruction)
 	{
 	case Instruction::STOP:
+		logTrace(_instruction, false);
```

Comment:
For instructions that do not write to memory such as `stop()`, the second argument is `false`

Reply:
I'm wondering - shouldn't we use some utility to infer this fact from the opcode itself?

Reply:
Sounds reasonable to me. More concretely, would it be a static look up table per opcode or an annotation to the opcode itself?

Reply:
I think `SemanticInformation::memory` would be suitable. It is a slight problem since we cannot test this function here anymore, but I guess it's better to use constants here all the time.

Reply:
Updated to make use of `SemanticInformation::memory`. If there are any mistaken assumptions, the fuzzer would report a bug which we could later investigate.